### PR TITLE
Eng 982 small updates

### PR DESCRIPTION
**Pull request by joewhite101**

### CODE REVIEW

File: vuepress/docs/next/tutorials/ecr/ecr-how-microservices-connects-to-apps.md [hunk 1]
```diff
@@ -1,66 +1,66 @@
...
-4. If both the EntandoPlugin custom resource and the EntandoAppPluginLink are deployed correctly, the APIs of the microservice will be available from the same domain of the 
-EntandoApp, making it possible reach those APIs from the EntandoApp using relative urls
+4. If both the EntandoPlugin custom resource and the EntandoAppPluginLink are deployed correctly, the APIs of the microservice will be available from the same domain of the EntandoApp, making it possible reach those APIs from the EntandoApp using relative urls
 
 This is the standard flow when no other micorservice with a given name is already available in the EntandoApp namespace.
 
-If such condition is not true and there's already microservice with the same name of the one described in the bundle, the ECR will connect directly the EntandoApp to
-the existing microservice by generating and deploying the required EntandoAppPluginLink as in the step 3 above. 
+If there is already microservice with the same name of the one described in the bundle, the ECR will connect directly the EntandoApp to
```
Comment (nshaw) [inline vuepress/docs/next/tutorials/ecr/ecr-how-microservices-connects-to-apps.md:16]: Suggestion: If there is an existing microservice with the same name as the one described in the bundle, the ECR will connect the EntandoApp to
the existing microservice by generating and deploying the required EntandoAppPluginLink per step 3 above.

File: vuepress/docs/next/tutorials/ecr/ecr-how-microservices-connects-to-apps.md [hunk 2]
```diff
@@ -1,66 +1,66 @@
...
+Check the [Kubernetes naming conventions](#k8s-name-conv) section for  rules on how to compose your microservice name. Here are some example of valid names:
 
 - `organization.microservice-name.version`: entando.custom-microservice.v1
 - `organization-microservice-name-major.minor`: my-org-special-microservice-v-2.0
 - `SHA256(organization/name:version)`: 79982d02a615ac8c68c989e59069cff7ec52eb6e41418b71ad199a3857104bfc (SHA256 of my-organzation/my-app:v2.0)
 
-### Microservice ingress path clashing
+### Microservice Ingress Path Clashing
+
+In Entando 6.2 when a bundle is uninstalled and the microservice is unlinked from the EntandoApp, from the EntandoApp ingress the path to the plugin is not removed. This could be an issue when trying to link an EntandoApp with different versions of the same microservices.
```
Comment (nshaw) [inline vuepress/docs/next/tutorials/ecr/ecr-how-microservices-connects-to-apps.md:51]: Can we clarify this statement "from the EntandoApp ingress the path to the plugin is not removed." I'm not sure what the right language is.

File: vuepress/docs/next/tutorials/ecr/ecr-how-microservices-connects-to-apps.md (hunk 1 above)
Comment (Kerruba) [reply vuepress/docs/next/tutorials/ecr/ecr-how-microservices-connects-to-apps.md:16]: @nshaw yeah please go ahead with the suggestion :+1:

**Comment by Kerruba** [comment]
@nshaw please have a look at the new version of the ingress path clashing as well as some other small changes I've made to the 6.1 version of the docs (just a title thing, nothing major)

**Comment by nshaw** [comment]
> )



> please have a look at the new version of the ingress path clashing as well as some other small changes I've made to the 6.1 version of the docs (just a title thing, nothing major)

See https://github.com/entando/entando-docs/pull/159 for a minor refinement on this.

**Comment by Kerruba** [comment]
Ok I think we should start closing some PR which are all about the same topic what you think @nshaw ?

**Comment by nshaw** [comment]
@Kerruba , yes, whatever you and @joewhite101 think makes sense. I have the one minor PR which merges to this one but other than that I think you and Joe own the PRs that are overlapping.

**Pull request merged**